triggers.Dockerfile

```
@@ -17,9 +17,12 @@ WORKDIR /home/daml
 COPY --chown=daml daml.yaml .
 COPY --chown=daml src src
 COPY --chown=daml scripts scripts
-RUN daml build --output app.dar
+COPY --chown=daml triggers triggers
+RUN mkdir target
+RUN daml build --output /home/daml/target/supplychain.dar
+RUN daml build --project-root triggers --output /home/daml/target/triggers.dar
```

The working directory has already been set to `/home/daml`. Frankly, I would rather drop `target` as well, to simplify things, as in `ui.Dockerfile`.
```suggestion
RUN daml build --output target/supplychain.dar
RUN daml build --project-root triggers --output target/triggers.dar
```

---

Relative path is calculated from the project root, that is my experience.